pkg/controller/backup_controller_test.go

```diff
@@ -56,23 +56,24 @@ func (b *fakeBackupper) Backup(logger logrus.FieldLogger, backup *pkgbackup.Requ
 	return args.Error(0)
 }
 
-func TestProcessBackupNonProcessedItems(t *testing.T) {
+func TestProcessBackupProcessing(t *testing.T) {
```

Comment:
the intent of this test was to cover all the cases where a backup would *not* be run, hence the "NonProcessedItems" part of the name - ok with a name change but I'd try to convey that intent.

Reply:
Turns out the name should not change, it's all unprocessed cases. Changing.